src/coreclr/interpreter/compiler.cpp

```diff
@@ -3500,6 +3500,16 @@ int InterpCompiler::GenerateCode(CORINFO_METHOD_INFO* methodInfo)
                         m_pLastNewIns->SetDVar(m_pStackPointer[-1].var);
                         m_ip++;
                         break;
+                    case CEE_SIZEOF:
+                    {
+                        CORINFO_CLASS_HANDLE clsHnd = ResolveClassToken(getU4LittleEndian(m_ip + 1));
```

Comment:
It seems it would make sense to add CHECK_STACK(1);

Reply:
I see, I thought CHECK_STACK was "make sure this many items are on the stack"

Reply:
Ah, I guess I am wrong in this case, I haven't realized the token is not on the stack.